dev-client/src/screens/SitesScreen/components/EmptySiteMessage.tsx

```diff
@@ -14,35 +14,68 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
-import {Trans} from 'react-i18next';
 
-import {BulletList} from 'terraso-mobile-client/components/BulletList';
-import {LocationIcon} from 'terraso-mobile-client/components/icons/LocationIcon';
+import {useTranslation} from 'react-i18next';
+import {StyleSheet} from 'react-native';
+
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+
+import {TranslatedParagraph} from 'terraso-mobile-client/components/content/typography/TranslatedParagraph';
+import {Icon} from 'terraso-mobile-client/components/icons/Icon';
 import {Text, View} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {convertColorProp} from 'terraso-mobile-client/components/util/nativeBaseAdapters';
+import {useIsOffline} from 'terraso-mobile-client/hooks/connectivityHooks';
+import {OfflineMessageBox} from 'terraso-mobile-client/screens/LocationScreens/components/soilId/messageBoxes/OfflineMessageBox';
 
 export const EmptySiteMessage = () => {
+  const {t} = useTranslation();
+  const isOffline = useIsOffline();
+
   return (
     <View px="17px">
-      <Text variant="body1">
-        <Trans
-          i18nKey="site.empty.info"
-          components={{
-            bold: <Text bold />,
-          }}
-        />
+      {isOffline ? <OfflineMessageBox message={t('site.offline')} /> : <></>}
+
+      <Text bold mt={isOffline ? 5 : undefined}>
+        <TranslatedParagraph i18nKey="site.empty.info" />
       </Text>
 
-      <BulletList
-        data={[1, 2, 3]}
-        renderItem={i => (
-          <Text variant="body1" color="text.primary">
-            <Trans
-              i18nKey={`site.empty.bullet_${i}`}
-              components={{icon: <LocationIcon />}}
-            />
-          </Text>
-        )}
-      />
+      <View style={styles.view}>
+        <MaterialCommunityIcons
```

Comment:
Question: Any chance MaterialCommunityIcons is a superset of MaterialIcon? If so, could we use that in our custom `Icon` component? It just feels nicer to have one unified way to refer to icons if possible.

Reply:
They appear to be separate. @expo/vector-icons includes a few different icons sets.